### PR TITLE
Update periodic Job

### PR DESCRIPTION
**Pull request by aojea**
update the periodic job with the new changes on the e2e testing
- new ha configuration
- kubernetes 1.19 version
- new go version

### CODE REVIEW

File: .github/workflows/test_periodic.yml [hunk 1]
```diff
@@ -110,7 +110,7 @@ jobs:
       fail-fast: false
       matrix:
         target:
-          - shard: shard-network
+          - shard: shard-conformance
```
[the remark this answers is not in the document]
Comment (aojea) [reply .github/workflows/test_periodic.yml:113]: it runs the conformance + network tests, ~300 tests in parlallel (20 workers).
network tests only test functionality, we need to understand how ovn-kube perform with workloads (adding/removing pods, reescheduling pods), and if affects any of the conformance.

File: .github/workflows/test_periodic.yml [hunk 2]
```diff
@@ -120,7 +120,7 @@ jobs:
     needs: [build, k8s]
     env:
       JOB_NAME: "${{ matrix.target.shard }}-${{ matrix.ha.name }}"
-      KIND_HA: "${{ matrix.ha.enabled }}"
+      OVN_HA: "${{ matrix.ha.enabled }}"
```
[the remark this answers is not in the document]
Comment (aojea) [reply .github/workflows/test_periodic.yml:123]: The 3 etcd cluster created for the Kubernetes HA control plane was literally starving all resources of the job, and we are not really interested in test kubernetes HA, for us is important to test OVN HA, that is orthogonal to it.
I've modified the deployment so we can choose between a 1 master ovn installation and a HA ovn installation (with multiple master and db pods)

**Comment by aojea** [comment]
This is a follow up from this
https://github.com/ovn-org/ovn-kubernetes/pull/1672

**Comment by danwinship** [comment]
My questions before were more about the fact that those changes don't seem to have anything at all to do with "bump dual stack e2e tests to 1.19". If this PR is doing more than just that, then can you update the commit message / PR description to indicate everything it's doing and why?

**Comment by aojea** [comment]
> My questions before were more about the fact that those changes don't seem to have anything at all to do with "bump dual stack e2e tests to 1.19". If this PR is doing more than just that, then can you update the commit message / PR description to indicate everything it's doing and why?

done

**Comment by aojea** [comment]
@danwinship can we marge this to see the results with kubernetes 1.19?
Right now ovn does not work for dual-stack, all the jobs time out, we had previously the same sympton due to network policies dropping all traffic, some of the failures show
```
2020-09-25T07:27:59.0014810Z {"msg":"FAILED [sig-network] NetworkPolicy [LinuxOnly] NetworkPolicy between server and client should enforce policy to allow traffic only from a pod in a different namespace based on PodSelector and NamespaceSelector [Feature:NetworkPolicy]","total":96,"completed":64,"skipped":4560,"failed":1,"failures":["[sig-network] NetworkPolicy [LinuxOnly] NetworkPolicy between server and client should enforce policy to allow traffic only from a pod in a different namespace based on PodSelector and NamespaceSelector [Feature:NetworkPolicy]"]}
```

since some fixes for dual stack are only 1.19 and were not backported, I prefer to start to investigate from 1.19 only.

**Comment by aojea** [comment]
:+1:

**Pull request merged**